resources/aws/securitygroup.go

```diff
@@ -0,0 +1,77 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+var (
+	portsToOpen []int = []int{
+		// SSH
+		22,
+		// Insecure apiserver
+		80,
+		// apiserver
+		443,
```

Comment:
Using 6443 was a temporary solution for #70 which IMO should be disregarded, but I wasn't aware that kubernetesd is using that too. Will add it.